docker/bookkeeper/Dockerfile

```diff
@@ -7,37 +7,35 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-
 FROM openjdk:8-jre-alpine
-MAINTAINER Arvind Kandhare [arvind.kandhare@emc.com]
+
+MAINTAINER Pravega Team
 
 RUN apk add --update \
     bash \
     wget \
     unzip \
-&&  rm -rf /var/cache/apk/* \
-&&  mkdir -p /opt \
-&&  cd /opt
-
-ENV BOOKKEEPER_VERSION=4.5.0 \
-    ZOOKEEPER_VERSION=3.5.3-beta
-
-RUN wget http://central.maven.org/maven2/org/apache/bookkeeper/bookkeeper-server/$BOOKKEEPER_VERSION/bookkeeper-server-$BOOKKEEPER_VERSION-bin.tar.gz \
-&&  tar xvzf  bookkeeper-server-$BOOKKEEPER_VERSION-bin.tar.gz \
-&&  mv bookkeeper-server-$BOOKKEEPER_VERSION /opt/bookkeeper
+    && rm -rf /var/cache/apk/*
 
-# zookeeper-3.5.3-beta.tar.gz is actually not compressed even though it is published as .tar.gz, so not using -z flag to untar
-RUN wget http://www.apache.org/dist/zookeeper/zookeeper-$ZOOKEEPER_VERSION/zookeeper-$ZOOKEEPER_VERSION.tar.gz \
-&&  tar xvf zookeeper-$ZOOKEEPER_VERSION.tar.gz \
-&&  mv zookeeper-$ZOOKEEPER_VERSION /opt/zookeeper
-
-ENV bookiePort 3181
+ARG BOOKKEEPER_VERSION=4.7.3
+ARG ZOOKEEPER_VERSION=3.5.4-beta
 
+ENV bookiePort=3181
 EXPOSE $bookiePort
 
-# all execute-able bin, download from apache BK and ZK
+RUN mkdir -p /opt \
+    && cd /opt \
+    && wget -q https://archive.apache.org/dist/bookkeeper/bookkeeper-${BOOKKEEPER_VERSION}/bookkeeper-server-${BOOKKEEPER_VERSION}-bin.tar.gz \
```

Comment:
aren't we checking GPG signatures ?

see the official BK image for 4.7 branch:
https://github.com/apache/bookkeeper/blob/branch-4.7/docker/Dockerfile

Reply:
Thanks, I will make the changes and check GPG and SHA.